pkg/vulnsrc/vulnerability/vulnerability.go

```diff
@@ -44,6 +44,27 @@ func getSeverity(details map[string]types.VulnerabilityDetail) types.Severity {
 	return types.SeverityUnknown
 }
 
+func getVendorSeverity(details map[string]types.VulnerabilityDetail) types.VendorSeverity {
+	vs := make(types.VendorSeverity)
+	for _, source := range sources {
```

Comment:
This `sources` should be different from `getSeverity` because some vendors use the same value as NVD. We don't have to store such a severity in DB.

Reply:
What should the sources be then?